content/en/docs/concepts/architecture/cgroups.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Cgroup V2
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+On Linux, {{< glossary_tooltip text="control groups" term_id="cgroup" >}}
+are used to constrain resources that are allocated to processes.
+
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} and the
+underlying container runtime need to interface with control groups to enforce
+[resource mangement for pods and
+containers](/docs/concepts/configuration/manage-resources-containers/) and set
+resources such as cpu/memory requests and limits.
+
+There are two versions of cgroups in linux: cgroupv1 and cgroupv2. Cgroupv2 is
+the new generation of the cgroup API.
+
+<!-- body -->
+
+
+## Cgroup version 2 {#cgroup-v2}
+{{< feature-state for_k8s_version="v1.25" state="stable" >}}
+
+Cgroup v2 is the next version of the cgroup Linux API. Cgroup v2 provides a
+unified control system, which provides enhanced resource management
+capabilities.
+
+The new version offers several improvements over cgroup v1, some of these improvements are:
+
+- cleaner and easier to use API with a unified hierarchy
+- safe sub-tree delegation to containers
+- newer features like Pressure Stall Information
+- enhanced accounting and isolation across multiple resources
+  - accounting for network memory
```

Comment:
Single bullet looks odd. Add more or remove this line

Reply:
Added a few more :)